config/samples/arcadia_v1alpha1_datasource.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: arcadia.kubeagi.k8s.com.cn/v1alpha1
 kind: Datasource
 metadata:
   name: datasource-sample
-  namespace: kubebb-system
+  namespace: arcadia-system
 spec:
-  url: my-minio.kubebb-system.svc.cluster.local:9000
+  url: my-minio.arcadia-system.svc.cluster.local:9000
```

Comment:
hi @zqq454224016  use namespace `arcadia` is fine. do not need to use `arcadia-system`

Reply:
Because arcadia has been deployed in this namespace

Reply:
You mean `minio` ? We all use same  namespace `arcadia`.If it is not,pls change it.